src/components/SchemaViewer.svelte

```diff
@@ -1,12 +1,11 @@
 <script>
-  import throttle from "just-throttle";
   import SchemaNode from "./SchemaNode.svelte";
+  import FilterBox from "./FilterBox.svelte";
 
   export let app;
   export let nodes = [];
-  let filterText = "";
   let nodesWithVisibility;
-  const filterTextChanged = () => {
+  const filterTextChanged = (filterText = "") => {
```

Comment:
I don't think this function needs a default value?

```suggestion
  const filterTextChanged = (filterText) => {
```

Reply:
This function is called inside SchemaViewer for the initial render. If I remove the default value it throws variable undefined error.